create-quasar/templates/app/quasar-v2/ts-vite/BASE/src/env.d.ts

```diff
@@ -7,3 +8,14 @@ declare namespace NodeJS {
     VUE_ROUTER_BASE: string | undefined;
   }
 }
+
+declare global {
```

Comment:
It would be better to move this into Quasar own types, so we have control of them
There are many `.d.ts` files for which we already add an import to make TS consider them, eg. `shim-icon-set` or `shim-lang` into `ui/types`

Could you move this in a `shim-jsx.d.ts` file and make the build script reference them?
You can find the build script into `ui/build/build.types.js`, other imports around line 561

Keep `any` for `style` for now, until we have the chance to dig deeper